vortex-serde/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rust-version = { workspace = true }
 
 [dependencies]
 ahash = { workspace = true }
+arrow-array = { workspace = true }
```

Comment:
where does this come from?

Reply:
not needed anymore, moved back to a dev dependency